inc/integrations/class-pico.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * WP Irving integration for Pico.
+ *
+ * @package WP_Irving
+ */
+
+namespace WP_Irving\Integrations;
+
+use WP_Irving\Components\Component;
+use Pico_Setup;
+use Pico_Widget;
+
+/**
+ * Class to integrate Pico with Irving.
+ */
+class Pico {
+
+	/**
```

Comment:
The rest of these integrations use a singleton pattern, though it looks like our `load_integrations()` function is instantiating a new class for each integration instead of getting a static instance. Can we fix this real quick?

Reply:
Ah, that's why I dropped the singleton stuff from this class, I didn't realize that was the intention. I'll push an update.

Reply:
Added a singleton and updated.